stack/indexer/deb/build_package.sh

```diff
@@ -50,6 +50,15 @@ build_deb() {
             base_cmd+="--reference ${reference}"
         fi
         ../base/generate_base.sh -s ${outdir} -r ${revision} ${base_cmd}
+    else
+        version=$(cat ${current_path}/../../../VERSION)
+        basefile="${outdir}/wazuh-indexer-base-${version}-${revision}-linux-x64.tar.xz"
+        if test -f "${basefile}"; then
+            echo "Building using base file: ${basefile}"
+        else
+            echo "Did not find expected Wazuh Indexer base file: ${basefile} in output path. Exiting..."
+            exit 0
```

Comment:
Returning 0 means that everything went well. This should be changed to a non-zero value

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/2462/commits/526b7d8ea905b0a6ece4df4f1d7fafa2ea6ae3f9